rustls/src/webpki/verify.rs

```diff
@@ -464,39 +466,130 @@ impl From<webpki::Error> for CertRevocationListError {
     }
 }
 
-static ECDSA_SHA256: SignatureAlgorithms = &[webpki::ECDSA_P256_SHA256, webpki::ECDSA_P384_SHA256];
-
-static ECDSA_SHA384: SignatureAlgorithms = &[webpki::ECDSA_P256_SHA384, webpki::ECDSA_P384_SHA384];
-
-static ED25519: SignatureAlgorithms = &[webpki::ED25519];
+type SignatureAlgorithms = &'static [&'static dyn webpki::SignatureVerificationAlgorithm];
```

Comment:
I don't think we should have this, makes it harder to understand the provenance (for example, how the webpki dependency is leaking into our public API).

Reply:
removed